lib/doorkeeper/oauth/token.rb

```diff
@@ -26,7 +26,7 @@ def from_basic_authorization(request)
 
         def token_from_basic_header(header, pattern)
           encoded_header = token_from_header(header, pattern)
-          token, _ = decode_basic_credentials(encoded_header)
+          token, = decode_basic_credentials(encoded_header)
```

Comment:
I can rewrite `decode_basic_credentials` to return only what is necessary. What do you think?